examples/ssh/iperf3.js

```diff
@@ -101,7 +122,26 @@ function validate() {
 * @documentation This procedure is used for retrieving iperf3 results
 */
 function get_status() {
-    execute()
-        .then(D.success)
-        .catch(failure);
-}
\ No newline at end of file
+    var serverIndex = 0;
+    function tryNextServer() {
```

Comment:
What is the purpose of this function? 
Just loop the severs and parametrize the conf

Reply:
in the case of one server is busy we jump to the next server to test the perf

Reply:
this section is removed, the check will be for every command server by server